Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -4,8 +4,17 @@ import QuartzCore
 
 class LayoutManager: NSLayoutManager {
 
+    var blockquoteBorderColor: UIColor = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
+    var blockquoteBackgroundColor = UIColor(red: 0.91, green: 0.94, blue: 0.95, alpha: 1.0)
```

Comment:
I don't think this needs to go for the initialiser when we can have a reasonable default value. None of the UIKit default components force you to set a default value, but they all allow customisation by the app.